Gemfile

```diff
@@ -24,6 +23,7 @@ group :test, :development do
   gem 'pry-byebug'
   gem 'rb-readline'
   gem 'rubocop', '~> 0.47.1', require: false
+  gem 'sqlite3'
```

Comment:
so the database config file tells it which adapter to use. In local development you can do a `bundler --without docker` to exclude the pg gem, though having both won't hurt anything besides some added complexity in compiling the pg gem.  For heroku and docker, it excludes the test and development groups so sqlite doesn't get installed